constants.js

```diff
@@ -13,7 +14,8 @@ const platformMapping = {
   ios: 'ios-swift',
   react: 'react',
   reactnative: 'react-native',
-  angular: 'angular'
+  angular: 'angular',
```

Comment:
Do we have different mappings for video and chat?
Because in video we don't have `angular` yet.